contracts/contracts/vault/VaultCore.sol

```diff
@@ -23,11 +23,8 @@ import "./VaultInitializer.sol";
 contract VaultCore is VaultInitializer {
     using SafeERC20 for IERC20;
     using StableMath for uint256;
-    // max signed int
+    /// @dev max signed int
     uint256 internal constant MAX_INT = 2**255 - 1;
```

Comment:
As long as we are cleaning up this section, we could change the `2**255 - 1` to `type(int256).max`

Reply:
agreed 👍

Reply:
Solidity won't allow `type(int256).max` to be set to an unsigned int `uint256`

Reply:
Just needs the cast so it's explicit. `uint256(type(int256).max)`